lib/cts/cases/cli.rb

```diff
@@ -270,6 +270,10 @@ def show(*args)
       end
     end
 
+    option :non_indexed, type: :boolean,
+           desc: 'Narrow down the scope of reindex to the cases which have not been indexed yet.'
+    option :size, aliases: 's', type: :numeric, default: 5000,
+            desc: 'Number of cases to for indexes. [5000]'
```

Comment:
I think there's a word missing, not sure what - limit or something? `Number of cases to for `

Reply:
changed the wording